core/domain/html_validation_service_test.py

```diff
@@ -1238,7 +1238,7 @@ def test_parsable_as_xml(self):
             True)
 
     def test_convert_svg_diagram_tags_to_image_tags(self):
-        test_case = {
+        test_cases = [{
```

Comment:
I suggest actually splitting these into their own test cases (it's generally better practice to have tests be as specific to the behaviors they're testing as possible to make it easier to verify that the test is correct). Parametrization is a nice way to check that different conditions for the same behavior yield the same outcome, but in this case I think that we're actually testing a few different behaviors.

Reply:
Done